templates/compose/excalidraw.yaml

```
@@ -0,0 +1,21 @@
+# documentation: https://docs.excalidraw.com/docs
+# slogan: Virtual whiteboard for sketching hand-drawn like diagrams 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)